geomstats/geometry/poincare_polydisk.py

```diff
@@ -102,11 +97,10 @@ class PoincarePolydiskMetric(ProductRiemannianMetric):
         https://epubs.siam.org/doi/pdf/10.1137/15M102112X
     """
 
-    def __init__(self, n_disks, default_coords_type="extrinsic"):
+    def __init__(self, n_disks):
         self.n_disks = n_disks
-        list_metrics = []
-        for i_disk in range(n_disks):
-            scale_i = (n_disks - i_disk) ** 0.5
-            metric_i = HyperboloidMetric(2, default_coords_type, scale_i)
-            list_metrics.append(metric_i)
+        base_metric = HyperboloidMetric(2)
+        list_metrics = [
+            (n_disks - i_disk) ** 0.5 * base_metric for i_disk in range(n_disks)
```

Comment:
Can we double check this @YannCabanes - we need to use the factor which scales the metric itself, rather than the distance. I suspect it should not have the `** 0.5`.

Reply:
Hello @johnharveymath and @lpereira95,
Yes, we have to decide which convention we are willing to use.
I was using the convention multiplying the distance by the scale parameter, which means multiplying the `metric_matrix` by `scale ** 2`.
It seems that you would prefer to multiply the `metric_matrix` by `scale` and therefore the distance by `scale ** 0.5`.
Both conventions are fine for me, so I will let you both decide.

Reply:
If you choose to multiply the `metric_matrix` by `scale`, then you can remove `** 0.5`.